release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -68,6 +68,12 @@ With this release, you can now install {op-system} to iSCSI boot devices. Multip
 [id="ocp-4-16-installation-and-update_{context}"]
 === Installation and update
 
+[id="ocp-4-16-installation-and-update-aws-capi_{context}"]
+==== Cluster API replaces Terraform for AWS installations
+In {product-title} {product-version}, the installation program uses Cluster API instead of Terraform to provision cluster infrastructure during installations on AWS. There are several additional required permissions as a result of this change. For more information, see xref:../installing/installing_aws/installing-aws-account.adoc#installation-aws-permissions_installing-aws-account[Required AWS permissions for the IAM user].
```

Comment:
```suggestion
In {product-title} {product-version}, the installation program uses Cluster API instead of Terraform to provision cluster infrastructure during installations on AWS. There are additional required permissions as a result of this change. For more information, see xref:../installing/installing_aws/installing-aws-account.adoc#installation-aws-permissions_installing-aws-account[Required AWS permissions for the IAM user].
```